en/docs/develop/components/service.md

```diff
@@ -21,7 +21,7 @@ Choreo defines endpoints by combining port binding, protocol, endpoint name, net
 | Name | A unique identifier for the endpoint within the service component. |
 | Port | The network port on which the endpoint is accessible. |
 | Type | The communication protocol used by the endpoint (e.g., HTTP, HTTPS, gRPC, etc.). |
-| Network Visibility | The level of visibility granted to the endpoint. Determines the level of visibility granted to the endpoint, with the following options: <ul><li>Project: the endpoint can only be accessed by other components within the same project.</li><li>Organization: the endpoint can be accessed by any component within the same organization, but not by components outside of the organization.</li><li>Public: the endpoint can be accessed by any component, regardless of its location or organization.</li></ul> |
+| Network Visibility | The level of visibility granted to the endpoint. Determines the level of visibility granted to the endpoint, with the following options: <ul><li>Project: the endpoint can only be accessed by other components within the same project.</li><li>Organization: the endpoint can be accessed by any component within the same organization, but not by components outside of the organization.</li><li>Public: the endpoint can be accessed by any client, regardless of its location or organization.</li></ul> |
```

Comment:
Suggestion to improve clarity and avoid passive voice:
Determines the level of visibility of an endpoint. Possible values are:
- Project: Allows components within the same project to access the endpoint.
- Organization: Allows any component within the same organization to access the endpoint but restricts access to components outside the organization.
- Public: Allows any client to access the endpoint, regardless of location or organization.

Reply:
Fixed with https://github.com/wso2/docs-choreo-dev/pull/837/commits/2035102fbe0abb353e98116d176dbadfdc6a0cb5